setup.py

```diff
@@ -10,6 +10,14 @@ def list_scripts():
     script_list = [f'{os.path.join(script_dir, f)}' for f in os.listdir(script_dir) if f.startswith('lstchain_')]
     return script_list
 
+entry_points = {}
+entry_points['console_scripts'] = [
+    'calc_camera_calibration = lstchain.tools.calc_camera_calibration:main',
+    'onsite_create_drs4_pedestal_file = scripts.onsite.onsite_create_drs4_pedestal_file:main',
+    'onsite_create_calibration_file = scripts.onsite.onsite_create_calibration_file:main',
+    'lstchain_data_create_pedestal_file = scripts.lstchain_data_create_pedestal_file:main'
```

Comment:
the scripts starting with "lstchain_" should be covered by l.10 of this file, right?

Reply:
Hi @rlopezcoto, I don't understand the question. But, indeed one can take out from here the lstchain scripts and keep only the onsite as command line

Reply:
I meant that they were included already in the script list, but I saw from one of your previous comments that your intention was actually to make them "runnable" by the users without including the `.py`

Reply:
They are already runnable, installed on your system when you install lstchain, and you don't need to call `python ***`. But I see that the `.py` naming really is confusing 😅

Reply:
I know @vuillaut, but believe me, it is better not to have the .py

Reply:
No no it's ok I agree (see below) 👍